lib/once.js

```diff
@@ -1,9 +1,9 @@
 import on from './on';
 import off from './off';
 
-function once(el, event, run, capture = false) {
-  on(el, event, e => {
-    off(el, event);
+function once(selector, event, run, capture = false) {
+  on(selector, event, e => {
+    off(selector, event);
```

Comment:
Unrelated to this PR but as per docs, the capture should be passed on to `off` too.